uber/templates/jobs/job_renderer.html

```diff
@@ -77,7 +77,7 @@
                                     .append(
                                         $('<td></td>').append(
                                             shift.worked === {{ c.SHIFT_UNMARKED }} ? renderWorked(shift.id) : ('<i>' + shift.worked_label + '</i>')))
-                                    .append(shift.worked === {{ c.SHIFT_UNMARKED }} ? '' : renderRating(shift, $('<td></td>')));
+                                    .append(shift.worked !== {{ c.SHIFT_WORKED }} ? '' : renderRating(shift, $('<td></td>')));
```

Comment:
This is the place where we were always rendering the rating form instead of only doing it for shifts marked as worked.